edit/auth/azuread.vue

```diff
@@ -126,6 +129,18 @@ export default {
         this.$set(this.model, key, ENDPOINT_MAPPING[endpoint][key].replace(TENANT_ID_TOKEN, this.model.tenantId));
       });
     },
+
+    getNewApplicationSecret() {
+      const applicationSecretOrId = this.model.applicationSecret || this.applicationSecret;
+
+      // The application secret comes back as an ID from steve API and this indicates
+      // that the current application secret isn't new
+      if (!applicationSecretOrId.includes('cattle-global-data')) {

```

Comment:
It looks like this needs to be inverted; this is including the applicationSecret in the authconfig PUT request only when it's an ID rather than never including when it's an ID

Reply:
Yeah, you're right. I must've messed it up when I moved things around after my first test. Good catch.